balancer-design.typ

```diff
@@ -36,14 +36,20 @@ In Rust, packages are called "crates". The Balancer and Harness is split into mu
 			
 			if(value.keys().contains("version")) { row.push(value.version) }
 			else if(value.keys().contains("path")) { row.push(value.path) }
-			else if(value.keys().contains("git")) { row.push(value.git) }
+			else if(value.keys().contains("git") and value.keys().contains("rev")) { 
+				row.push(
+					"git: " + value.git + "\n" +
+					"rev: " + value.rev
+				) 
+			}
+			else if(value.keys().contains("git")) { row.push("git: " + value.git) }
+			else if(value.keys().contains("rev")) { row.push("rev: " + value.rev) }
```

Comment:
nit: technically, `rev` is only valid if `git` is present, so this extra branch is unnecessary